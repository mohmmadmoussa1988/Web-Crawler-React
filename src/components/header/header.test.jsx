import React from 'react';
import {configure,shallow} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import Header from './header';
import toJson from 'enzyme-to-json';

configure({adapter:new Adapter()});
describe(`<Header />`,()=>{

    it('should render Header',()=>{
        const wrapper = shallow(<Header />);
        expect(toJson(wrapper)).toMatchSnapshot();
    })


})