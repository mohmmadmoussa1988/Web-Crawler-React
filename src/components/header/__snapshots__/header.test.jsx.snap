// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should render Header 1`] = `
<styled.header>
  <styled.div>
    <Logo />
  </styled.div>
</styled.header>
`;
